src/engine/networking/RemoteSync.js

```diff
@@ -460,13 +460,17 @@ class RemoteSync
 	 * It synchronizes with a remote object which has the same
 	 * shared id.
 	 * @param {Object3D} object
-	 * @param {string} id - shared id.
 	 * @param {boolean} recursive - recursively adds children if true
 	 */
-	addSharedObject(object, recursive)
+	addSharedObject(object, id, recursive)
```

Comment:
this is how `three-network` comes out of the box, actually. Previously I created a small shared object id generator, because I couldn't think of a good reason to leave this to the user, then I think I nixed it to eliminate potential issues when toubleshooting one of the _numerous_ issues with networking I ran into:). I like your idea further down about making it a top-of-file constant.